### PR TITLE
fix(ui): Fix table styling for components and projects

### PR DESCRIPTION
**Pull request by maierthomas**
this pull request fixes the table styling (column width, column overlap) for:
- linked projects, Project, EDIT Mode
- linked releases, Projects, EDIT Mode
- linked releases, Components, EDIT Mode

closes eclipse/sw360#112

### CODE REVIEW

**Comment by adrian-evo** [comment]
Tested and works as expected.

File: frontend/sw360-portlet/src/main/webapp/html/projects/ajax/linkedProjectsAjax.jsp [hunk 1]
```diff
@@ -23,36 +22,28 @@
...
         <td width="32%">
             <input type="hidden" value="${projectLink.id}" name="<portlet:namespace/><%=Project._Fields.LINKED_PROJECTS%><%=ProjectLink._Fields.ID%>">
-            <label class="textlabel stackedLabel" for="projectName">Project name</label>
             <input id="projectName" type="text" class="toplabelledInput" placeholder="Enter project"
                    value="<sw360:out value="${projectLink.name}"/>" readonly onclick="window.location='<sw360:DisplayProjectLink projectId="${projectLink.id}" bare="true"/>'"/>
         </td>
         <td width="32%">
             <input type="hidden" value="${projectLink.version}" name="<portlet:namespace/><%=ProjectLink._Fields.VERSION%>">
-            <label class="textlabel stackedLabel" for="projectVersion">Project version</label>
             <input id="projectVersion" type="text" class="toplabelledInput" placeholder="No project version"
```
Comment (alexbrdn) [inline frontend/sw360-portlet/src/main/webapp/html/projects/ajax/linkedProjectsAjax.jsp]: This and similar fields all have the class `toplabelledInput`, but now you removed their labels. Does it have any implications other than a little inconsistency in code?

Comment (maierthomas) [reply frontend/sw360-portlet/src/main/webapp/html/projects/ajax/linkedProjectsAjax.jsp]: indeed they could lead to confusion. I removed the ´toplabelledInput´ class attributes
